### PR TITLE
Topology-toolbar-filter feature

### PR DESCRIPTION
**Pull request by hemantsaini-7**
**Description:**

- Topology-toolbar-filter feature | Topology-automation

**Jira Id:**
     - [ODC-6675](https://issues.redhat.com/browse/ODC-6675)

**Pre-conditions/setup:**

**Checks for approving Epic scenarios Automation PR:**

- [x] Execute the @to-do tagged gherkin scripts manually
- [x] Convert the @to-do gherkin scripts to cypress automation scripts
- [x] Once scripts are automated, replace tag @to-do with @epic-number
- [x] Execute the scripts in Remote cluster

**Refactoring criteria for approving Automation PR:**

- [ ] update the test cases count in [Automation status confluence Report](https://docs.jboss.org/display/ODC/Automation+Status+Report)

**Execution Commands:**
Example:
```
    export NO_HEADLESS=true && export CHROME_VERSION=$(/usr/bin/google-chrome-stable --version)
    BRIDGE_KUBEADMIN_PASSWORD=YH3jN-PRFT2-Q429c-5KQDr
    BRIDGE_BASE_ADDRESS=https://console-openshift-console.apps.dev-svc-4.11-042801.devcluster.openshift.com/
    export BRIDGE_KUBEADMIN_PASSWORD
    export BRIDGE_BASE_ADDRESS
    oc login -u kubeadmin -p $BRIDGE_KUBEADMIN_PASSWORD
    oc apply -f ./frontend/integration-tests/data/htpasswd-secret.yaml
    oc patch oauths cluster --patch "$(cat ./frontend/integration-tests/data/patch-htpasswd.yaml)" --type=merge
    ./test-cypress.sh -p topology
```
**Screen shots**:

Topology-toolbar-filter.feature
image

**Browser conformance**:

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

File: frontend/packages/topology/integration-tests/features/topology/topology-toolbar-filter.feature [hunk 1]
```diff
@@ -8,8 +8,10 @@ Feature: Topology Toolbar Filter Group
               And user is at Add page
 
 
-        @regression @to-do
+        @regression
         Scenario: Topology filter by resource: T-13-TC01
+            Given user has created workload "nodejs-ex-git-d" with resource type "Deployment"
+              And user is at Add page
             Given user has created workload "nodejs-ex-git-dc" with resource type "Deployment Config"
```
Comment (SparshKesari) [inline frontend/packages/topology/integration-tests/features/topology/topology-toolbar-filter.feature:15]: We can replace 'Given' with 'And'

Comment (hemantsaini-7) [reply frontend/packages/topology/integration-tests/features/topology/topology-toolbar-filter.feature]: updated

**Comment by sanketpathak** [comment]
Tests are green
/approve

**Comment by SparshKesari** [comment]
Pr only affects E2E tests
/label qe-approved
/label docs-approved
/label px-approved

**Pull request merged**